cli/tests/symlink_to_subdir

```diff
@@ -0,0 +1 @@
+subdir/
```

Comment:
Add this path to .gitattributes and add a "symlink=dir" attribute.

Reply:
Done